ignite/cmd/plugin.go

```diff
@@ -360,6 +364,130 @@ func NewPluginUpdate() *cobra.Command {
 	}
 }
 
+func NewPluginAdd() *cobra.Command {
+	cmdPluginAdd := &cobra.Command{
+		Use:  "add [path]",
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return fmt.Errorf("must provide a plugin path")
+			}
+			chain, err := newChainWithHomeFlags(cmd)
+			s := cliui.New(cliui.WithStdout(os.Stdout))
+
+			if err != nil {
+				return err
+			}
+			nodePath := chain.ConfigPath()
+			nodePath = path.Dir(nodePath)
+
+			confPath, err := pluginsconfig.LocateDefault(nodePath)
+			if err != nil {
+				return err
+			}
+			var conf pluginsconfig.Config
+			f, err := os.Open(confPath)
+			if err != nil {
+				return err
+			}
+			err = conf.Decode(f)
+			if err != nil {
+				return err
+			}
```

Comment:
I think you can simplify all these by just calling `parseLocalPlugins()`

Reply:
Well, on my side I don't use `chain.ConfigPath()` to get the config file path, but rather like `parseLocalPlugins()`, I consider the config path is passed via the `--path` flag or else it's the working dir. 

And unless I'm wrong this is exactly what `newChainWithHomeFlags(cmd)` is doing under the hood, in addition to many other things that are useless in this context. So finally let's just read the `--path` flag and stop call `newChainWithHomeFlags(cmd)`. Is this OK for you?

Reply:
> And unless I'm wrong this is exactly what `newChainWithHomeFlags(cmd)` is doing under the hood,

Yes, I believe this is true.

Reply:
this does indeed work, I removed the unessesary calls to `newChainWithHomeFlags` and added a new return value to `parseLocalPlugins` `func parseLocalPlugins(rootCmd *cobra.Command) (cfg *pluginsconfig.Config, path string, err error)` this makes the persisting of the new config state much simpler.

Reply:
awesome